.github/workflows/ci.yaml

```diff
@@ -9,13 +9,11 @@ jobs:
     strategy:
       matrix:
         go:
-          - '=1.18'
-          - '^1.23'
+          - "^1.23"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go }}
-      - run: go test -v ./...
-      - run: cd v2 && go test -v ./...
+      - run: cd v3 && go test -v ./...
```

Comment:
This shouldn't replace v2, we should run CI on both.

Reply:
Wasn't sure on the best strategy here. We can't run v3 against 1.18, but I assume we still want to run v1/2 against it. I think we would need separate tests to do this. Let me know if you want this or if you're happy with just running all 3 versions against 1.23.

Reply:
You can just duplicate the job itself. Rename this job to `unit-tests-v2` and add a `unit-tests-v3`.